File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Delegate/DelegateSamples.cs
public delegate void FooType();

public partial class ParentType
{
    public delegate void NestedType();
}

public partial class ParentType
{
    public partial class AnotherNestedType
    {
        public delegate void DeeperNestedType();
    }
}

namespace FooNamespace
{
    public delegate void InNamespaceType();
}

namespace FooNamespace.BarNamespace
{
    public partial class ParentType
    {
        public delegate void NestedType();
    }
}

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Delegate/DelegateSamples.vb
Public Delegate Sub FooType()

Public Partial Class ParentType
    Public Delegate Sub NestedType()
End Class

Public Partial Class ParentType
    Public Partial Class AnotherNestedType
        Public Delegate Sub DeeperNestedType()
    End Class
End Class

Namespace FooNamespace
    Public Delegate Sub InNamespaceType()
End Namespace

Namespace FooNamespace.BarNamespace
    Public Partial Class ParentType
        Public Delegate Sub NestedType()
    End Class
End Namespace